### PR TITLE
Add a topic talk about jaeger topology

### LINKED ISSUE #424: Need a topic talk about jaeger topology
I try to use jaeger to generate topology with jaeger-ui.

And I found there are two entry about topology, one is "System Architecture" and other is "Deep Dependency Graph". I konw for open source version, System Architecutre read data in cassandra or elasticsearch, and Deep Dependency Graph only read data by search result. But, what is the different between these? Are they aim to solve differenct problem?  What is the road map about Topology? Maybe we need some a detailed document to talk about this?

I have read this issue before: 
https://github.com/jaegertracing/jaeger-ui/issues/547#issuecomment-600930999

### PR DESCRIPTION
**Pull request by salmon7**
## Which problem is this PR solving?
- Resolves #424

## Short description of the changes
- Add a detailed document to talk about jaeger topology.

### CODE REVIEW

File: content/docs/next-release/features.md [hunk 1]
```diff
@@ -55,3 +55,18 @@ from advanced features not available elsewhere, if your organization has already
...
+
+## Topology
+
+In production, the Dependencies from `System Architecture` is based on service data which stored in Cassandra or Elasticsearch, 
+while `Deep Dependency Graph` only read trace data by the search result. The main differences between them are:
+
+System Architecture view:  
+  * is built from one-hop connections between services. E.g. A - B - C implies that A and B, and B and C are connected, but says nothing about A and C
+  * The graph is displayed for all services in the system at once
+  * Only takes into accounts services, not their specific endpoints
```
Comment (objectiser) [inline content/docs/next-release/features.md:67]: "accounts" -> "account"

File: content/docs/next-release/features.md [hunk 2]
```diff
@@ -55,3 +55,18 @@ from advanced features not available elsewhere, if your organization has already
 using Zipkin libraries, you do not have to rewrite all that code. Jaeger provides backwards compatibility with Zipkin
 by accepting spans in Zipkin formats (Thrift, JSON v1/v2 and Protobuf) over HTTP. Switching from Zipkin backend is just a matter
 of routing the traffic from Zipkin libraries to the Jaeger backend.
+
+## Topology
+
+In production, the Dependencies from `System Architecture` is based on service data which stored in Cassandra or Elasticsearch,
```
Comment (objectiser) [inline content/docs/next-release/features.md:61]: Its not specific to production or any particular storage - the main point is that if using production setup then need to also run the spark job, so possibly:

"The 'System Architecture' is based on service dependencies, derived from span data reported to the Jaeger Collector. When using a production configuration, the dependencies are derived using a Spark job."

and then "The 'Deep Dependency Graph' is generated based on trace instances obtained by doing a search. The main differences between them are:"?

File: content/docs/next-release/features.md [hunk 3]
```diff
@@ -55,3 +55,18 @@ from advanced features not available elsewhere, if your organization has already
...
+while `Deep Dependency Graph` only read trace data by the search result. The main differences between them are:
+
+System Architecture view:  
+  * is built from one-hop connections between services. E.g. A - B - C implies that A and B, and B and C are connected, but says nothing about A and C
+  * The graph is displayed for all services in the system at once
+  * Only takes into accounts services, not their specific endpoints
+
+Deep Dependency Graph:  
+  * is built with transitive knowledge of end to end flows through the system. You can mouse over a particular edge in the graph and see all call paths passing through it
+  * is build on the notion of a "focal service" for which the graph is rendered, meaning that paths through the system not passing through the focal service are not displayed
```
Comment (objectiser) [inline content/docs/next-release/features.md:71]: "is built"

File: content/docs/next-release/features.md [hunk 4]
```diff
@@ -55,3 +55,18 @@ from advanced features not available elsewhere, if your organization has already
...
+  * is build at the level if individual endpoints (although service-only view is also supported)
```
Comment (objectiser) [inline content/docs/next-release/features.md:72]: "is built"

File: content/docs/next-release/features.md [hunk 5]
```diff
@@ -55,3 +55,18 @@ from advanced features not available elsewhere, if your organization has already
 using Zipkin libraries, you do not have to rewrite all that code. Jaeger provides backwards compatibility with Zipkin
 by accepting spans in Zipkin formats (Thrift, JSON v1/v2 and Protobuf) over HTTP. Switching from Zipkin backend is just a matter
 of routing the traffic from Zipkin libraries to the Jaeger backend.
+
+## Topology
```
Comment (yurishkuro) [inline content/docs/next-release/features.md:59]: Topology Graphs

**Comment by salmon7** [comment]
@yurishkuro @objectiser I have updated PR, please review again.

**Comment by yurishkuro** [comment]
Thanks for starting the PR.

**Pull request merged**